cmd/plugin-backend.go

```diff
@@ -11,15 +11,19 @@ import (
 )
 
 var (
-	portArg         = flag.Int("port", 0, "server port to listen on (default: 9443)")
-	certArg         = flag.String("cert", "", "cert file path to enable TLS (disabled by default)")
-	keyArg          = flag.String("key", "", "private key file path to enable TLS (disabled by default)")
-	featuresArg     = flag.String("features", "", "enabled features, comma separated")
-	staticPathArg   = flag.String("static-path", "", "static files path to serve frontend (default: './web/dist')")
-	configPathArg   = flag.String("config-path", "", "config files path (default: './config')")
-	pluginConfigArg = flag.String("plugin-config-path", "", "plugin yaml configuration")
-	logLevelArg     = flag.String("log-level", "error", "verbosity of logs\noptions: ['panic', 'fatal', 'error', 'warn', 'info', 'debug', 'trace']\n'trace' level will log all incoming requests\n(default 'error')")
-	log             = logrus.WithField("module", "main")
+	portArg                   = flag.Int("port", 0, "server port to listen on (default: 9443)\n9444 and 9445 reserved for other use")
+	certArg                   = flag.String("cert", "", "cert file path to enable TLS (disabled by default)")
+	keyArg                    = flag.String("key", "", "private key file path to enable TLS (disabled by default)")
+	featuresArg               = flag.String("features", "", "enabled features, comma separated")
+	staticPathArg             = flag.String("static-path", "", "static files path to serve frontend (default: './web/dist')")
+	configPathArg             = flag.String("config-path", "", "config files path (default: './config')")
+	pluginConfigArg           = flag.String("plugin-config-path", "", "plugin yaml configuration")
+	logLevelArg               = flag.String("log-level", "error", "verbosity of logs\noptions: ['panic', 'fatal', 'error', 'warn', 'info', 'debug', 'trace']\n'trace' level will log all incoming requests\n(default 'error')")
+	alertmanagerNameArg       = flag.String("alertmanager-name", "", "alertmanager name for acm mode")
```

Comment:
should we allow to provide the full URL rather than namespace and name?

```suggestion
	alertmanager       = flag.String("alertmanager", "", "alertmanager URL")
```

```
-alertmanager=https://alertmanager-main.openshift-monitoring.svc:9094
```

WDYT?

Reply:
Most recent commit changes to use URL parameters. Also updates the feature flag to 'acm-alerting' rather than 'acm'.